blogs/mariatta.ca.json

```
@@ -15,7 +15,8 @@
          "github": "mariatta",
          "linkedin": "mariatta",
          "instagram": "mariatta81",
-         "youtube": "mariattaw/videos"
+         "youtube": "mariattaw/videos",
+         "bluesky": "@mariatta.bsky.social"
```

```suggestion
         "bluesky": "@mariatta.ca"
```

Thank you! I've just updated my bluesky using my domain name.